### ISSUE TITLE
BWT for 0.3.0

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/community-services#3]
TODO:

1. Better wallet management (ie delete wallet files when deleted from config)
2. Instructions
3. Add custom broadcast command
4. rescan date for each xpub?
5. Test wallet descriptors
6. Endpoint authentication?
7. Add progress when rescanning
8. It doesn't work unless you input an xpub before starting
9. Decide whether to use bitcoind or proxy
10. Add health checks - bitcoind synced, bitcoind rescanning, etc
11. Handle pruning
12. Guess-and-check attack warning (in interface description)

**Comment by kn0wmad** [comment]
Is this still a "To Do"

**Comment by chrisguida** [comment]
Has anything changed?

**Comment by kn0wmad** [comment]
My understanding is we are abandoning this, so can we close it?

**Comment by ProofOfKeags** [comment]
It's not part of the 0.3 scope of work, but I don't think we are abandoning it completely.

**Comment by chrisguida** [comment]
If we package electrs, bwt may not have much of a use case.

We should research any use cases before abandoning

**Comment by MattDHill** [comment]
Decision made to go with electrs. If we decide to go with BWT in the future, we can track in another ticket.

**Comment by chrisguida** [comment]
If anyone wants to finish bwt, it's here: https://github.com/Start9Labs/bwt-wrapper
